src/Model/MethodGo.cs

```diff
@@ -551,7 +551,7 @@ public string AutorestError(string phase, string response = null, string paramet
                                  : string.Format("autorest.NewErrorWithError(err, \"{0}.{1}\", \"{2}\", {3}, \"{4}\")", PackageName, Owner, methodName, response, phase);
         }
 
-        public string ValidationError => $"validation.NewErrorWithValidationError(err, \"{PackageName}.{Owner}\",\"{Name}\")";
+        public string ValidationError => $"validation.NewError(\"{PackageName}.{Owner}\", \"{Name}\", err.Error())";

```

Comment:
>onError => $"validation.NewError(\"{PackageName}.{Owner}\", \"{Name}\", err.Error())"; [](start = 30, length = 86)

Nit: I think you can use both string interpolation and verbatim strings at the same time ( in the order  $@'') in case it helps with the escaping.